src/backend/src/services/work-packages.services.ts

```diff
@@ -281,8 +281,8 @@ export default class WorkPackagesService {
     blockedBy: WbsNumber[],
     expectedActivities: DescriptionBullet[],
     deliverables: DescriptionBullet[],
-    projectLead: number,
-    projectManager: number
+    projectLeadId: number,
```

Comment:
can you update the comment for this service method to use the right names as well